docs/docs/writing-hello-world.md

```diff
@@ -423,7 +366,7 @@ EnclaveHost mockHost = EnclaveHost.load("com.r3.conclave.sample.enclave.ReverseE
 mockHost.start(null, null);
 ```
 
-Conclave will detect that the enclave class is on the classpath and will start the enclave in mock mode. You
+Conclave detects the enclave class on the classpath and starts the enclave in mock mode. You
 can obtain the enclave instance using the
 [`EnclaveHost.mockEnclave`](api/-conclave%20-core/com.r3.conclave.host/-enclave-host/get-mock-enclave.html) property.
 
```

Comment:
Would it make sense to point the developer to a sample mock unit test?

https://github.com/R3Conclave/conclave-tutorials/blob/HEAD/hello-world/enclave/src/test/java/com/r3/conclave/sample/enclave/MockTest.java